### PR TITLE
Gdcps 776

### PR DESCRIPTION
**Pull request by mhockenbury**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -41,13 +42,37 @@ var server = restify.createServer({});
     });
 
     server.use(function requireConsumerParam(req, res, next) {
-        if(!req.params.consumer) {
+        if (req.url.indexOf('help') > -1 || req.url.indexOf('api-docs') > -1) { // don't require consumer params for swagger
```
Comment (akshaykarthik) [inline index.js]: can we do 
`if(blah) return next()`
`if(blah2) return next()`
`return next()`

that way adding new conditions doesn't need fiddling around with else ifs and braces

File: index.js [hunk 2]
```diff
@@ -41,13 +42,37 @@ var server = restify.createServer({});
...
+            next();
+        } else if(!req.params.consumer) {
             next(new restify.InvalidArgumentError('Consumer not passed in'));
         } else {
             next();
         }
     });
 
+    server.get('/api-docs', function (req, res, next) {
+        res.send(require('./swagger/swagger.json'));
```
Comment (akshaykarthik) [inline index.js]: i know that require's cache will take care of keeping this in memory but can we make that explicit?

Comment (mhockenbury) [inline index.js]: like move it up to a top level `var swaggerJSON = require(...);` with the other requires?

File: index.js (hunk 1 above)
Comment (mhockenbury) [inline index.js]: Yea, I hated this conditional setup, will refactor this.

File: index.js (hunk 2 above)
Comment (akshaykarthik) [inline index.js]: yeah

**Pull request merged**